optimum/exporters/openvino/model_configs.py

```diff
@@ -4511,15 +4526,33 @@ def generate_dummy_inputs(self, framework: str = "pt", **kwargs):
         return dummy_inputs
 
 
-@register_in_tasks_manager("ernie4_5", *["text-generation", "text-generation-with-past"], library_name="transformers")
-class ErnieOpenVINOConfig(TextDecoderWithPositionIdsOnnxConfig):
```

Comment:
ernie4_5 was never officially supported (added in transformers 4.54) and export is currently not working as expected : looks like issue is coming from `rotate_half` https://github.com/huggingface/transformers/blob/v4.54.0/src/transformers/models/ernie4_5/modeling_ernie4_5.py#L93  cc @openvino-dev-samples I removed it for now and we can open a new PR to add it back (with proper patching)